onnxruntime/server/converter.h

```diff
@@ -13,16 +14,16 @@
 namespace onnxruntime {
 namespace server {
 
-onnx::TensorProto_DataType MLDataTypeToTensorProtoDataType(const onnxruntime::DataTypeImpl* cpp_type);
+onnx::TensorProto_DataType MLDataTypeToTensorProtoDataType(ONNXTensorElementDataType cpp_type);
 
 // Convert MLValue to TensorProto. Some fields are ignored:
 //   * name field: could not get from MLValue
 //   * doc_string: could not get from MLValue
 //   * segment field: we do not expect very large tensors in the prediction output
 //   * external_data field: we do not expect very large tensors in the prediction output
 // Note: If any input data is in raw_data field, all outputs tensor data will be put into raw_data field.
-common::Status MLValueToTensorProto(const OrtValue& ml_value, bool using_raw_data,
-                                    std::unique_ptr<onnxruntime::logging::Logger> logger,
+common::Status MLValueToTensorProto(Ort::Value& ml_value, bool using_raw_data,
+                                    std::shared_ptr<spdlog::logger> logger,
```

Comment:
can this shared_ptr be passed by const ref? do you have to increment the ref count here by passing it by value?